x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/lens_attributes.ts

```diff
@@ -495,7 +496,7 @@ export class LensAttributes {
       if (qFilter.query?.bool?.should) {
         const values: string[] = [];
         let fieldName = '';
-        qFilter.query?.bool.should.forEach((ft: PersistableFilter['query']['match_phrase']) => {
+        qFilter.query?.bool.should.forEach((ft: any) => {
```

Comment:
Curious as to why this change is necessary?

Reply:
Same question - is it possible to be more explicit?

Reply:
Fixed